app/models/application_record.rb

```diff
@@ -1,3 +1,20 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  @@month_names = {
```

Comment:
do you have to use class variables here? just so you can grab it in any class that inherits from ApplicationRecord? I am not super comfortable with class variables... do you think you could turn this into a constant instead? it looks like that would be sufficient. 
let me know what you think